### ISSUE TITLE
Should the `Query.One()` method imply a limit of one?

### ISSUE THREAD

**Issue by mheffner** [go-ozzo/ozzo-dbx#87]
When using the `Query.One` method to populate a single row from the database (like below), it would seem to me like the implication is that it will only select a single row by adding a `LIMIT 1` to the query. However, it appears to run the full query without a limit and drops all but the first row. This can be quite expensive if you weren't expecting it.

Should `Query.One()`  add a `LIMIT 1`? If not, I would recommend this be very clearly advertised in the documentation. 

```
q := db.NewQuery("SELECT id, name FROM users ORDER BY id")

// populate the first row into a User struct
err = q.One(&user)
fmt.Println(user.ID, user.Name)
```

**Comment by kolkov** [comment]
Good catch. `Query.One()` does run the full query and discards extra rows, which can be expensive on large tables. Automatically adding `LIMIT 1` would be a reasonable optimization.

This has been sitting for a while. Is this still something you'd want to see addressed? We'll close in a couple of weeks if no response, but it's a valid improvement we might pick up.

**Comment by mheffner** [comment]
I'm not using the project anymore.
